#!/bin/sh -e
##
## 90_debian_alsaconf_idmode.dpatch by Thomas Hood
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use Debian-specified GID and mode for ALSA devices.

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad alsa-utils-1.0.8/alsaconf/alsaconf.in /tmp/dpep.YfAcTT/alsa-utils-1.0.8/alsaconf/alsaconf.in
--- alsa-utils-1.0.8/alsaconf/alsaconf.in	2005-02-18 21:40:15.000000000 +0100
+++ /tmp/dpep.YfAcTT/alsa-utils-1.0.8/alsaconf/alsaconf.in	2005-02-18 21:40:15.000000000 +0100
@@ -104,8 +104,8 @@
 do_legacy_only=0
 use_modinfo_db=0
 alsa_uid=0
-alsa_gid=0
-alsa_mode=0666
+alsa_gid=29
+alsa_mode=0660
 legacy_probe_card=""
 LOGFILE=""
 TESTSOUND="@TESTSOUND@"
